Add balance-tx scheme

j-mueller opened the pull request.

j-mueller committed:
Add balance-tx scheme

Anviking commented on CIP-0013/CIP-0013.md:
```diff
@@ -100,17 +111,27 @@ For brevity, essential in many Internet contexts, `poolticker`  must be supporte
+
+#### Interpretation of `payload`:
+
+The "payload" parameter is the JSON representation of an [ExportTx](https://github.com/input-output-hk/plutus/blob/master/plutus-contract/src/Plutus/Contract/Wallet.hs#L113-L120) value, gzipped and Base-64 encoded using the URL-safe alphabet
```
@j-mueller The referenced datatype contains a [`signatories `](https://github.com/input-output-hk/plutus/blob/5e9d0d9046b0ca7e3cfbef99661f67599122d5a2/plutus-contract/src/Plutus/Contract/Wallet.hs#L118) field, but I believe Matthias just confirmed with you that such a field would be un-needed? So I suppose it should be updated.

j-mueller replied:
I'm currently updating the export code on our side, I'll add a full example to this PR and remove the link

nikolaglumac replied:
Thanks! Please keep us posted 🙏

j-mueller replied:
@anviking changed, let me know if you need more!

nikolaglumac replied:
@Anviking are you ok with this now?

Anviking replied:
> @Anviking are you ok with this now?

The `signatories` field was just an obvious discrepancy I noticed. Now that it has been adjusted, I believe both Jann and Matthias would better be able to tell whether things are _completely_ right! (E.g. things internal to the payload — and I think they are discussing in tmp_adrestia-plutus-coordination)

Anviking replied:
> I'll add a full example to this PR and remove the link

@j-mueller FYI: The link is still there, and because the file has been updated, the selection is a few lines off.

If you were going to remove it, that sounds reasonable, as you already refer to the CWB API in a section above anyway:

> The JSON payload of this URI shall be passed by Daedalus, or other wallets that handle the link format, directly to the [CWB balance transaction](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/balanceTransaction) API as its request body

which I guess should be enough.

nikolaglumac replied:
> I'll add a full example to this PR and remove the link

@j-mueller please let us know once you sort this out 🙏

nikolaglumac replied:
@j-mueller any news on this?

j-mueller committed:
Update example

j-mueller committed:
Update

j-mueller commented:
@Anviking @nikolaglumac  I updated the description and the example to match the format that `cardano-wallet` currently uses and that we've implemented in the PAB.

nikolaglumac commented:
Thanks @j-mueller!
@tomislavhoracek feel free to merge this one 🙏
